packages/presets/client/src/index.ts

```diff
@@ -133,6 +133,8 @@ export const preset: Types.OutputPreset<ClientPresetConfig> = {
       avoidOptionals: options.config.avoidOptionals,
       documentMode: options.config.documentMode,
       skipTypeNameForRoot: options.config.skipTypeNameForRoot,
+      onlyOperationTypes: options.config.onlyOperationTypes,
+      onlyEnumTypes: options.config.onlyEnumTypes,
```

Comment:
I think this is not correct...?

https://the-guild.dev/graphql/codegen/plugins/typescript/typescript#onlyenums

```suggestion
      onlyEnums: options.config.onlyEnums,
```

Reply:
- https://github.com/dotansimha/graphql-code-generator/pull/10280